index.md

```diff
@@ -42,6 +42,18 @@
 
 ---
 
+| | Pacman.store |
+| - | - |
+| **Description** | pkg cache for Pacman + ISOs |
+| ***Init&Run*** | `ipfs-cluster-follow cluster.pacman.store run --init cluster.pacman.store/default.json` or `ipfs-cluster-follow cluster.pacman.store run --init cluster.pacman.store/mdns_enabled.json` |
```

Comment:
This results in loading a configuration from `http://127.0.0.1:8080/ipns/cluster.pacman.store/default.json` but cluster.pacman.store is not a dnslink address.

I think the right commands (unless dnslink is setup) should be:

`ipfs-cluster-follow cluster.pacman.store run --init https://cluster.pacman.store/default.json` (with the full https url).

This however fails because the certificate for that domain is not valid.

Reply:
cluster.pacman.store is a dnslink address:

```
 $ dig TXT +short cluster.pacman.store @8.8.8.8
"dnslink=/ipns/12D3KooWMSwL73UBcixpGiwVn2i2Ku88TPeELdjfQEKWiRWQ4Exi"
"1|https://ipfs.io/ipns/cluster.pacman.store/"
```

Also two people already joined the cluster that way, apart from my test follower. So it should work.

Reply:
Hmm, I expected it should be `dig TXT +short _dnslink.cluster.pacman.store`. But things seem to work then, I just gave up too quickly because I did not realize it was ipns-resolving after that.